docs/usage_guide/advanced_usage/modin_logging.rst

```diff
@@ -53,3 +53,39 @@ Disable Modin logging like so:
   LogMode.disable()
 
   # User code goes here
+
+Debugging from user defined functions:
+
+
+.. warning:: 
+  When attempting to use Modin logging in user defined functions that execute in workers for logging lower-level operators
```

Comment:
Check indents here. There should be 4 spaces like we have in modin/docs/flow/modin/core/dataframe/algebra.rst:L62.